app/controllers/sign_up/recovery_codes_controller.rb

```diff
@@ -6,10 +6,7 @@ class RecoveryCodesController < ApplicationController
     before_action :confirm_has_not_already_viewed_recovery_code, only: [:show]
 
     def show
-      if user_session.delete(:first_time_recovery_code_view).present?
-        @show_progress_bar = true
-      end
-
+      user_session.delete(:first_time_recovery_code_view)
```

Comment:
this was added so that the user can only view the recovery code once (since it is regenerated on each page load) See https://github.com/18F/identity-idp/commit/5748444953f1a5760d557f8c02ee85bcb197470c#diff-45f98c11ad4485219f731483536b7d10 for ref